core/app/[locale]/(default)/(auth)/register/page.tsx

```diff
@@ -26,18 +32,25 @@ export default async function Register() {
     notFound();
   }
 
-  const { addressFields, customerFields, reCaptchaSettings } = registerCustomerData;
-  const reCaptcha = await bypassReCaptcha(reCaptchaSettings);
+  const { addressFields, customerFields } = registerCustomerData;
+  // const reCaptcha = await bypassReCaptcha(reCaptchaSettings);
 
   return (
-    <div className="mx-auto mb-10 mt-8 text-base lg:w-2/3">
-      <h1 className="my-6 text-4xl font-black lg:my-8 lg:text-5xl">{t('heading')}</h1>
-      <RegisterCustomerForm
-        addressFields={addressFields}
-        customerFields={customerFields}
-        reCaptchaSettings={reCaptcha}
-      />
-    </div>
+    <SignUpSection
+      action={registerCustomer}
+      fields={[
```

Comment:
Sure, I just need a form skeleton then. Do we have any?

Reply:
And could we maybe save that for a future PR? That seems like scope we could potentially cut seeing as how main currently doesn't support this.

Reply:
We've been building this functionality as part of the release by default, having said that we can track this in a different issue, unless @migueloller feel otherwise.

Reply:
I think that if we make it streamable we should make a skeleton component to use as fallback. We could start without streamable support, which would make this page have to `await` the fields in the server, increasing TTFB, which isn't a big deal for a sign up page. That being said, I still think we should have a skeleton for the dynamic form so we _can_ make fields streamable and be consistent

Reply:
Agreed that using a skeleton and streaming is a better UX, I just think that since the baseline main version doesn't have this behavior we should prioritize it last given all of the other work we need to get through.

Reply:
Yeah, I agree. We can start without streamable props support and without a skeleton for now and introduce that later.

Reply:
I will track this because I think we definitely want to include.

Reply:
Sounds good. I think at this point given how close we are to the deadline, we're focused on feature parity with `main` and not wanting to spend much time on new functionality or behaviors not in `main`. Since `main` doesn't stream or have a skeleton for the sign up form, it's ok for us to ship it without it. We all agree that we should track it to include eventually, though!